_targets.R

```diff
@@ -5,19 +5,61 @@ library(tibble)
 suppressPackageStartupMessages(library(dplyr))
 
 options(tidyverse.quiet = TRUE)
-tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr", "rnaturalearth", "cowplot", "lubridate"))
+tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr",
+                            "rnaturalearth", "cowplot", "lubridate",
+                            "leaflet", "leafpop", "htmlwidgets"))
 
 # Load functions needed by targets below
 source("1_fetch/src/find_oldest_sites.R")
 source("1_fetch/src/get_site_data.R")
 source("2_process/src/tally_site_obs.R")
+source("2_process/src/summarize_targets.R")
 source("3_visualize/src/map_sites.R")
 source("3_visualize/src/plot_site_data.R")
+source("3_visualize/src/plot_data_coverage.R")
+source("3_visualize/src/map_timeseries.R")
 
 # Configuration
 states <- c('WI','MN','MI','IL','IN','IA')
 parameter <- c('00060')
 
+# Static branching targets
+# PULL SITE DATA
+mapped_by_state_targets <- tar_map(
+  values = tibble(state_abb = states,
+                  state_plot_files = sprintf('3_visualize/out/timeseries_%s.png', state_abb)),
+  names = state_abb,
+  unlist = FALSE,
+  tar_target(nwis_inventory,
+             {
+               oldest_active_sites %>%
+                 filter(state_cd == state_abb)
+             }
+  ),
+  tar_target(nwis_data,
+             get_site_data(
+               nwis_inventory,
+               state_abb,
+               parameter)
+  ),
+  # Insert step for tallying data here
+  tar_target(tally,
+             tally_site_obs(
+               site_data = nwis_data
+             )
+  ),
+  # Insert step for plotting data here
+  tar_target(timeseries_png,
+             plot_site_data(
+               out_file = state_plot_files,
+               site_data = nwis_data,
+               parameter = parameter
+             ),
+             format = 'file'
+  )
+)
```

Comment:
I like the single lines for readability but for me, it can make it less readable and more overwhelming because there are so many lines. I think we will have to develop some norms once we are all working in the same repos, but my personal preference for `tar_map()` was to make it much shorter:

```r
# Define static branching before targets list
mapped_by_state_targets <-
  # Pull data for each state's oldest site
  tar_map(
    values = tibble(state_abb = states) %>%
      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
    tar_target(nwis_inventory, filter(oldest_active_sites, state_cd == state_abb)),
    tar_target(nwis_data, get_site_data(nwis_inventory, state_abb, parameter)),
    # Insert step for tallying data here
    tar_target(tally, tally_site_obs(nwis_data)),
    # Insert step for plotting data here
    tar_target(timeseries_png, plot_site_data(state_plot_files, nwis_data, parameter), format="file"),
    names = state_abb,
    unlist = FALSE
  )
```